frontend/src/employee-frontend/components/vasu/components/FollowupQuestion.tsx

```diff
@@ -117,6 +122,7 @@ const FollowupEntryEditor = React.memo(function FollowupEntryEditor({
     } else {
       onChange({
         ...entry,
+        id: uuid(),
```

Comment:
Not sure generating an `uuid` in the frontend is a good thing. There must be a reason why there's no other place in the codebase that does this 🤔 

I think at least PostgreSQL has been set up in a way that we use (maybe?) UUID v1 that are based on timestamp and mac address so that they cannot collide even theoretically.

Reply:
Is the `id` even used anywhere? It wasn't there before this commit but this commit adds no usages.

Reply:
The ID is needed to be able to edit the comment after it has been saved. I tried two approaches:
1. Generating the ID here in the frontend when creating an entry
2. Generating the ID in the service, and reloading the document afterwards

With 2. we increase the complexity of the code quite a bit and I got into a bunch of corner cases and timing issues. Would require a bit bigger refactor in order for this case to work correctly... So I went with 1. for now.